findGame/TipOnLoadingPage.js

```diff
@@ -7,40 +7,47 @@
-        "If you see someone break a rule you can report them by clicking on their ball, their name in chat or the report button on the scoreboard.",
-        "Don't get mad at your team! Instead, give them tips on how to improve.",
-        "Team tiles don't make you go faster if you're holding the flag.",
-        "When you're trapped in your end zone in a neutral flag game, pop yourself to reset the flag.",
+        "Check out the <a href='https://reddit.com/r/tagpro'>/r/TagPro subreddit</a> to read the news or hang out!",
+        "Meet TagPro players on our <a href='https://reddit.com/r/tagpro/wiki/mumble'>voice-chat Mumble servers</a>. Microphone not required!",
+        "If someone is being abusive to anyone, report them! Click on their ball, or their name in chat, or click Report on the scoreboard (Esc).",
+        "Don't lash out on your team. Be understanding and supportive <3",
+        "Team tiles don't make you go faster if you're the flag carrier (FC).",
+        "In a neutral flag game if you're trapped in your base, you can pop yourself to reset the flag.",
         "Be mindful of when your powerup is going to run out. Powerups last for 20 seconds.",
-        "Leave the rolling bomb powerup for your flag carrier if they are nearby.",
-        "When you are trying to grab the flag, use boosts, bombs, and powerups to disrupt the enemy defense.",
-        "You have 0.25 seconds of immunity after grabbing the flag. Use this to your advantage.",
-        "If you move slowly, you can change directions faster and have a better chance of juking enemies.",
-        "Don't guard an empty base! Return the flag.",
-        "Walls can be used to quickly change direction when you are trying to juke other players.",
-        "Have fun! If you're on a losing streak or getting frustrated then take a break."
+        "Leave the rolling bomb powerup for your flag carrier if they are near.",
+        "Use boosts, bombs and powerups to disrupt the enemy defense and grab the flag.",
+        "After grabbing the flag you have 0.25 seconds of immunity!",
+        "The slower you move, the quicker you can change direction to juke out the enemies.",
+        "Don't guard your empty base. It's often better to help chase the flag carrier.",
+        "Use walls to quickly change direction.",
+        "If you start to become frustrated take a break!"
+        "Press Enter to chat to everyone, or T to chat with your team."
+        "Don't stand still for 30 seconds or the game will kick you out, thinking you're away from keyboard (AFK)."
+        "If you wrongfully get reported, don't worry. It takes 8 reports in 24 hours for them to have an effect."
+        "Learn how to enhance your gameplay experience on our <a href='https://www.reddit.com/r/TagPro/wiki/modding'>wiki page</a>!"
+        "Save your favorite moments with the magical <a href='https://www.reddit.com/r/TagPro/wiki/tagpro_replays_extension'>Replays extension.</a>"
```

Comment:
While both modding and replays are super cool, we don't officially support these. Also this is aimed at newer players.

Please remove.